src/Symfony/Component/HttpKernel/EventListener/ProfilerListener.php

```diff
@@ -129,8 +130,7 @@ public function onKernelTerminate(TerminateEvent $event): void
             $this->profiler->saveProfile($this->profiles[$request]);
         }
 
-        $this->profiles = new \SplObjectStorage();
-        $this->parents = new \SplObjectStorage();
+        $this->reset();
```

Comment:
we don't need this call then, since reset is going to be called anyway

Reply:
I just trying to preserve the old logic.
I'll drop it if it's not necessary then